index.js

```diff
@@ -1,26 +1,118 @@
+    technologies: ['HTML', 'CSS', 'JS', 'Ruby'],
+    seeLive: 'https://jawad-nazari.github.io/',
+    sourceCode: 'https://github.com/Jawad-Nazari/Portfolio-Mobile-1',
+  },
+  {
+    projectTitle: 'Uber Navigation',
+    imageURL: './image/card4.svg',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries.',
+    technologies: ['HTML', 'CSS', 'JS', 'Ruby'],
+    seeLive: 'https://jawad-nazari.github.io/',
+    sourceCode: 'https://github.com/Jawad-Nazari/Portfolio-Mobile-1',
+  },
+];
 
-menuItem.forEach((link) => {
-  link.addEventListener('click', () => {
-    hamburger.classList.remove('open');
-    menuOpen = false;
-    nav.classList.remove('open');
-    document.body.classList.remove('stop-scrolling');
+document.querySelectorAll('.btn').forEach((occurence) => {
+  const id = occurence.getAttribute('id');
+  const popupCard = `
+  <div class="project1" id="project1">
+        <div class="popupTop">
+            <div class="projectPopupTitle">
+              <div class="titlePopup">
+                <h3 class="comp-title">${data[id].projectTitle}</h3>
+                <img
+                id="popupClose${id}"
+                class="popupClose"
+                src="./icons/close.svg"
+                alt="closeBtn"
+                />
+              </div>
+              <ul class="work-title">
+                    <li class="client">CANOPY</li>
+                    <li><img src="./image/Counter.svg" alt="dot" /></li>
+                    <li class="dim">Back End Dev</li>
+                    <li><img src="./image/Counter.svg" alt="dot" /></li>
+                    <li class="dim">2015</li>
+                </ul>
+            </div>
+            <img class="projectImage" src=${data[id].imageURL} alt="projectImage">
+          <div class="popupDown">
+          <div class="project-desc popupDesc">${data[id].description}</div>
+            <div class="popupBtns">
+              <ul class="tags">
+                <li class="tag">${data[id].technologies[0]}</li>
+                <li class="tag">${data[id].technologies[1]}</li>
+                <li class="tag">${data[id].technologies[2]}</li>
+                <li class="tag">${data[id].technologies[3]}</li>
+              </ul>
+              <hr>
+              <div class="actionBtnPopup">
+               <a class="btn btnPopup" href=${data[id].seeLive} target='blank'>
+               See Live <img class="seeAll" src='./Icons/Icon.svg' alt="seeLive"></a>
```

Comment:
- [ ] Please 🙏🏽 edit the src attribute path to avoid broken links the correct path is `src='./icons/Icon.svg'`.